tutorials/examples/cov2d_simulation.py

```diff
@@ -35,71 +30,65 @@
 # Set the total number of images generated from the 3D map
 num_imgs = 1024
 
-# Set the number of 3D maps
-num_maps = 1
-
-# Set the signal-noise ratio
-sn_ratio = 1
+# Set the noise variance and build the noise filter
+# It might be better to select a signal noise ratio
+# and initial noise inside the Simulation class.
+noise_var = 1.3957e-4
+noise_filter = ScalarFilter(dim=2, value=noise_var)
 
 # Specify the CTF parameters
-pixel_size = 5                   # Pixel size of the images (in angstroms).
+pixel_size = 5*65/img_size       # Pixel size of the images (in angstroms)
 voltage = 200                    # Voltage (in KV)
-defocus_min = 1.5e4              # Minimum defocus value (in angstroms).
-defocus_max = 2.5e4              # Maximum defocus value (in angstroms).
+defocus_min = 1.5e4              # Minimum defocus value (in angstroms)
+defocus_max = 2.5e4              # Maximum defocus value (in angstroms)
 defocus_ct = 7                   # Number of defocus groups.
 Cs = 2.0                         # Spherical aberration
 alpha = 0.1                      # Amplitude contrast
 
 logger.info('Initialize simulation object and CTF filters.')
 # Create filters
-filters = [RadialCTFFilter(pixel_size, voltage, defocus=d, Cs=2.0, alpha=0.1)
+ctf_filters = [RadialCTFFilter(pixel_size, voltage, defocus=d, Cs=2.0, alpha=0.1)
            for d in np.linspace(defocus_min, defocus_max, defocus_ct)]
 
-# Load the map file of a 70S Ribosome and downsample the 3D map to desired resolution.
-# The downsampling should be done by the internal function of sim object in future.
-# Below we use alternative implementation to obtain the exact result with Matlab version.
+# Load the map file of a 70S Ribosome
 logger.info(f'Load 3D map and downsample 3D map to desired grids '
             f'of {img_size} x {img_size} x {img_size}.')
 infile = mrcfile.open(os.path.join(DATA_DIR, 'clean70SRibosome_vol_65p.mrc'))
 vols = Volume(infile.data)
-vols = vols.downsample(img_size)
+vols = vols.downsample(img_size)*1.0e3
```

Comment:
Why?

Reply:
Ok. Might be better to normalize it by the maximum intensity instead, then. Otherwise it looks like `1e3` is a magic number of some sort.

Reply:
I can change it to a different value but it requires to regenerate all binary data again. It might be ok since it is only a unit test and we have many other hard coded values already.

Reply:
This is not a unit test, it's a simulation. What data do we have to regenerate?

Reply:
Ok. I got a little confused with unit tests and tutorial example. I tended to use the same parameters for initializing the simulation object except higher resolution and more images for tutorial example. I will take a look again.

Reply:
The map density have be normalized by the maximum value in new commit 638c8fb.